roles/acm_setup/tasks/main.yml

```diff
@@ -96,7 +96,69 @@
         availabilityConfig: "{{ hub_availability }}"
         disableHubSelfManagement: "{{ hub_disable_selfmanagement }}"
 
-- name: "Wait for pods to be Running"
+- name: "Wait for search-postgres deployment"
+  k8s_info:
+    kind: Deployment
+    namespace: "{{ hub_namespace }}"
+    name: search-postgres
+  register: deployment_info
+  until:
+    - deployment_info is defined
+    - deployment_info.resources is defined
+    - deployment_info.resources | length > 0
+  retries: 180
+  delay: 10
+
+- name: "Get workers information"
+  community.kubernetes.k8s_info:
+    kind: Node
+  register: node_info
+  no_log: true
+
+- name: "Configure hugepages on postgres containers"
+  vars:
+    has_hugepages: "{{ node_info.resources |
+                    selectattr('status.allocatable.hugepages-1Gi', 'match', '^[1-9]') |
```

Comment:
Since we have made a requirement that when hugepages are used, they must be 2Mi. Can we remove this selection?